tuf_conformance/test_static_repositories.py

```diff
@@ -6,8 +6,8 @@
 from tuf_conformance.simulator_server import StaticServer
 
 static_repos = []
-for static_dir in os.listdir(os.path.join("tuf_conformance", "static_data")):
-    if os.path.isdir(os.path.join("tuf_conformance", "static_data", static_dir)):
+for static_dir in os.listdir(StaticServer.data_dir):
+    if os.path.isdir(os.path.join(StaticServer.data_dir, static_dir)):
         static_repos.append(static_dir)
```

Comment:
I suppose this whole thing could be a `StaticServer.test_names() -> list[str]`. I think I will do that still to keep the path handling in a single place